ci/integration_tests/src/tests/cw_core_test.rs

```diff
@@ -0,0 +1,406 @@
+    let account = key
+        .public_key()
+        .account_id(&chain.cfg.chain_cfg.prefix)
+        .unwrap();
+
+    // add item:
+    let res = create_dao(
+        chain,
+        Some(account.to_string()),
+        "exc_items_create_dao",
+        account.to_string(),
+    );
+
+    assert!(res.is_ok());
+    let dao = res.unwrap();
+
+    let res = chain
+        .orc
+        .query(
+            "cw_core",
+            "exc_items_get",
+            &cw_core::msg::QueryMsg::GetItem {
+                key: "meme".to_string(),
+            },
+        )
+        .unwrap();
+    let res: GetItemResponse = serde_json::from_slice(res.data.unwrap().value()).unwrap();
+
+    assert_eq!(res.item, None);
+
+    chain
+        .orc
+        .execute(
+            "cw_core",
+            "exc_items_set",
+            &cw_core::msg::ExecuteMsg::ExecuteAdminMsgs {
+                msgs: vec![CosmosMsg::Wasm(cosmwasm_std::WasmMsg::Execute {
+                    contract_addr: dao.addr.clone(),
+                    msg: to_binary(&cw_core::msg::ExecuteMsg::SetItem {
+                        key: "meme".to_string(),
+                        addr: "foobar".to_string(),
+                    })
+                    .unwrap(),
+                    funds: vec![],
+                })],
+            },
+            &chain.key,
+        )
+        .unwrap();
+
+    let res = chain
+        .orc
+        .query(
+            "cw_core",
+            "exc_items_set",
+            &cw_core::msg::QueryMsg::GetItem {
+                key: "meme".to_string(),
+            },
+        )
+        .unwrap();
+    let res: GetItemResponse = serde_json::from_slice(res.data.as_ref().unwrap().value()).unwrap();
+
+    assert_eq!(res.item, Some("foobar".to_string()));
+
+    // remove item:
+    chain
+        .orc
+        .execute(
+            "cw_core",
+            "exc_items_rm",
+            &cw_core::msg::ExecuteMsg::ExecuteAdminMsgs {
+                msgs: vec![CosmosMsg::Wasm(cosmwasm_std::WasmMsg::Execute {
+                    contract_addr: dao.addr,
+                    msg: to_binary(&cw_core::msg::ExecuteMsg::RemoveItem {
+                        key: "meme".to_string(),
+                    })
+                    .unwrap(),
+                    funds: vec![],
+                })],
+            },
+            &chain.key,
+        )
+        .unwrap();
+
+    let res = chain
+        .orc
+        .query(
+            "cw_core",
+            "exc_items_rm",
+            &cw_core::msg::QueryMsg::GetItem {
+                key: "meme".to_string(),
+            },
+        )
+        .unwrap();
+    let res: GetItemResponse = serde_json::from_slice(res.data.as_ref().unwrap().value()).unwrap();
+
+    assert_eq!(res.item, None);
+}
+
+// #### InstantiateMsg #####
+
+#[test_context(Chain)]
+#[test]
+fn instantiate_with_no_admin(chain: &mut Chain) {
+    let key: SigningKey = chain.key.clone().try_into().unwrap();
```

Comment:
seeing this at the top of each test makes me think there should be an easier way to get this info?

Reply:
Great point. Ill fix this.